examples/examples_py_test.go

```diff
@@ -179,6 +179,7 @@ func TestRegress2534(t *testing.T) {
 }
 
 func TestRegress4457(t *testing.T) {
+	t.Skipf("TODO[pulumi/pulumi-aws#4957]")
```

Comment:
What's happening here?

Reply:
Ah wait, just noticed I linked the PR and not the issue

Reply:
corrected the link, thanks for noticing!